src/components/quiz/QuizPage.tsx

```diff
@@ -1,15 +1,23 @@
-import React from "react";
+import React, { useState } from "react";
 import { QuizQuestions } from "./QuizQuestions";
-import { UserDetails } from "./UserDetails";
+import { UserDetailsForm } from "./UserDetailsForm";
 import { Reward } from "./Reward";
 
+interface Stat{
+  userStatus: boolean;
+}
+
 export const QuizPage: React.FunctionComponent = () => {
-  return (
-    <div>
-      <h1>This is the Quiz Component</h1>
-      <UserDetails />
-      <QuizQuestions />
-      <Reward />
-    </div>
-  );
+
+  const [userStatus, setUserStatus] = useState(false);
+ 
+  if (userStatus===false) { 
+    return <UserDetailsForm status={userStatus} setStatus={setUserStatus} />;
+  } else {
+   
+    return <QuizQuestions />
+  }
+   return <Reward />
```

Comment:
Your indenting is a little messed up here!
```suggestion
  return <Reward />
```